assets/js/theme/common/product-details.js

```diff
@@ -70,6 +70,7 @@ export default class ProductDetails {
                 $input: $('[data-product-stock]', $scope),
             },
             $sku: $('[data-product-sku]'),
+            $upc: $('[data-product-upc]'),
```

Comment:
The data tag already exists https://github.com/bigcommerce/cornerstone/blob/0d95be52e150449f78aa4ea868a0ab5e8cd3185f/templates/components/amp/products/product-view-details.html#L30